strawberryfields/api_client.py

```diff
@@ -0,0 +1,101 @@
+from urllib.parse import urljoin
```

Comment:
Don't forget to add the copyright notice to the top of this file :slightly_smiling_face:

Reply:
Yep will add to all the files.